Gems/Atom/Feature/Common/Assets/Shaders/Outline/Outline.azsl

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <viewsrg.srgi>
+
+#include <Atom/Features/PostProcessing/FullscreenVertex.azsli>
+#include <Atom/Features/PostProcessing/FullscreenPixelInfo.azsli>
+
+ShaderResourceGroup PassSrg : SRG_PerPass
+{
+    // D32_FLOAT_S8X24_UINT (Format:15) depthStencil texture
+    Texture2D<float2> m_depthStencil;
```

Comment:
Is this needed here? I wasn't sure if we need to add this even though we are not using it explicitly.

Reply:
will remove if we don't need